torch/__init__.py

```diff
@@ -300,10 +300,69 @@ def set_default_dtype(d):
     _C._set_default_dtype(d)
 
 def set_deterministic(d):
-    r"""Sets a global flag to force all operations to use a deterministic
-    implementation if available. If an operation that does not have a
-    deterministic implementation is called while this setting is True, the
-    operation will throw a RuntimeError.
+    r"""Sets a global flag to force normally nondeterministic operations to
+    use a deterministic implementation, if available. Or if any of these
+    operations do not have a deterministic implementation, they will throw
+    a :class:`RuntimeError`.
+
+    .. warning::
+        This feature is a work in progress, so it does not affect every
```

Comment:
Instead of WiP maybe "prototype" would align better with our feature language. @gchanan

Reply:
To me, "prototype" means an early proof-of-concept implementation, only meant for the research phase of developing something, to be thrown out in the future and replaced by a user-facing implementation. I feel like that could make people think that it's not ready to be used yet.

Reply:
How do you feel about "Beta?"

Reply:
Yeah I think beta makes sense.